packages/lumx-react/src/components/user-block/UserBlock.tsx

```diff
@@ -110,17 +113,17 @@ export const UserBlock: Comp<UserBlockProps, HTMLDivElement> = forwardRef((props
             isLink && linkProps?.className,
         );
         if (isLink) {
-            return renderLink({ ...linkProps, linkAs, className: nameClassName }, name);
+            return renderLink({ ...linkProps, linkAs, className: nameClassName, ...nameProps }, name);
         }
         if (onClick) {
             return (
-                <button onClick={onClick} type="button" className={nameClassName}>
+                <button onClick={onClick} type="button" className={nameClassName} {...nameProps}>
                     {name}
                 </button>
             );
         }
         return <span className={nameClassName}>{name}</span>;
```

Comment:
Should we not forward the props here too ?

Reply:
indeed! I did not see it oops